NuGet.Config

```diff
@@ -2,6 +2,13 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 
 <configuration>
+  <packageRestore>
+    <!-- Currently, the repository's version of NuGet.exe and Visual Studio's version
+         fight over the format of project.lock.json because the one in the respository
+         is newer. To prevent that, turn off package restore. -->
```

Comment:
We just need to get NuGet updated so that Roslyn can restore cross platform, I'm working on it now.